source/assets/sass/visual-design/_theme.scss

```diff
@@ -3,9 +3,12 @@
 @use '../config/color/brand';
 
 html {
-  @include config.tokens(config.$scale);
-  @include config.tokens(config.$brand-colors);
-  @include config.tokens(config.$content-colors);
+  &,
+  &::selection {
+    @include config.tokens(config.$scale);
+    @include config.tokens(config.$brand-colors);
+    @include config.tokens(config.$content-colors);
+  }
 
```

Comment:
@mirisuzanne this is how I organized it and this is working locally.

Reply:
looks great